core/src/main/java/com/netflix/iceberg/ManifestReader.java

```diff
@@ -187,11 +165,6 @@ private void cacheChanges() {
   }
 
   CloseableIterable<ManifestEntry> entries(Collection<String> columns) {
-    if (entries != null) {
-      // if this reader is an in-memory list or if the entries have been cached, return the list.
-      return CloseableIterable.withNoopClose(entries);
```

Comment:
I think `CloseableIterable.withNoopClose` is no longer used by any other code.

Reply:
We may still find a use for it, so I think we should leave it.